### PR TITLE
4494 Add Rocky Linux OS for Hetzner cloud provider

### PR DESCRIPTION
**Pull request by Waseem826**
### What this PR does / why we need it
Support for Rocky Linux has been added for Hetzner cloud provider.

screenshot-localhost_8000-2022 05 30-15_25_47

### Which issue(s) this PR fixes

#4494 

### Release note

```release-note
Rocky Linux OS has been added for Hetzner cloud provider.
```

### CODE REVIEW

**Comment by Waseem826** [comment]
@maciaszczykm we need to hold this PR until @zreigz tests it because there are few things not working on backend.

File: src/app/node-data/template.html [hunk 1]
```diff
@@ -133,7 +138,7 @@
       <div fxLayout="row"
            fxLayoutGap="36px">
         <!-- Options for Ubuntu, CenOS, SLES and RHEL -->
```
Comment (KhizerRehan) [inline src/app/node-data/template.html:140]: kindly, add Rocky Linux in comment too

**Comment by zreigz** [comment]
only centos-8 is not supported, please don't remove centos-7 (please verify which version is used)

**Comment by Waseem826** [comment]
It is not possible in the UI to select CentOS 7 since you can only select CentOS but I guess you can provide custom image. I'm also not sure if you can identify centos-7 in the UI. So for now I’ve brought back the CentOS option.

**Pull request merged**